include/AST/Decl/StructDecl.hpp

```diff
@@ -71,6 +71,13 @@ class StructDecl final
     /// @return Returns the field count of this struct.
     size_t getFieldCount() const { return _numFields; }
 
+    size_t getNeededFieldCount()
```

Comment:
this won't work if the default arg is on the first field but the second field has no default